sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobErrorCode.java

```diff
@@ -375,6 +369,9 @@ public final class BlobErrorCode extends ExpandableStringEnum<BlobErrorCode> {
     public static final BlobErrorCode AUTHORIZATION_RESOURCE_TYPE_MISMATCH =
             fromString("AuthorizationResourceTypeMismatch");
 
+    /** Static value SnaphotOperationRateExceeded for BlobErrorCode. */
+    public static final BlobErrorCode SNAPHOT_OPERATION_RATE_EXCEEDED = fromString("SnaphotOperationRateExceeded");
+
```

Comment:
why did we un-deprecated?

Reply:
I think this was probably an oversight after generating. We're waiting on expanded customization abilities to handle deprecating this automatically.